jwql/utils/edb.py

```diff
@@ -0,0 +1,40 @@
+#! /usr/bin/env python
+"""Tests for the ``engineering_database`` module.
```

Comment:
@Johannes-Sahlmann Did you write this test module? I'm thinking it should either be deleted (if it was accidentally copied somehow) or else moved into the `tests/` directory

Reply:
I may be wrong but I think this copied code was used by @DanielKuebi  to investigate EDB access issues early on. @DanielKuebi can you confirm?

Reply:
confirm :) sorry forgot to delete